modal/_types.py

```diff
@@ -0,0 +1,17 @@
+import os
+from typing import TypeVar
+
+F = TypeVar("F")
+
+
+def typechecked(f: F) -> F:
+    # type checking function adds significant overhead when
+    # the typeguard.typechecked is applied, so only do it locally
+    # where it matters the most
+    is_local = not bool(os.environ.get("MODAL_IMAGE_ID", ""))
```

Comment:
i think you can use `config.get("image_id")` also

Reply:
nice!